keep/api/routes/dashboard.py

```diff
@@ -31,6 +34,41 @@ class DashboardResponseDTO(BaseModel):
 
 
 router = APIRouter()
+logger = logging.getLogger(__name__)
+
+
+def provision_dashboards(tenant_id: str):
+    dashboards_raw = json.loads(os.environ.get("KEEP_DASHBOARDS", "[]"))
+    logger.info(
+        "Provisioning Dashboards", extra={"num_of_dashboards": len(dashboards_raw)}
+    )
+    dashboards_to_provision = [
+        DashboardCreateDTO.parse_obj(dashboard) for dashboard in dashboards_raw
+    ]
+    for dashboard in dashboards_to_provision:
+        logger.info(
+            "Provisioning Dashboard",
+            extra={"dashboard_name": dashboard.dashboard_name},
+        )
+        try:
+            create_dashboard_db(
```

Comment:
if dashboard already exists? it will create over and over?

Reply:
No. There's a unique constraint for the name of the dashboard.
`__table_args__ = (
        UniqueConstraint(
            "tenant_id", "dashboard_name", name="unique_dashboard_name_per_tenant"
        ),
    )`